pipeline/projects/knoedler/__init__.py

```diff
@@ -1305,15 +1320,16 @@ class ModelReturn(ModelSale):
 	helper = Option(required=True)
 	make_la_person = Service('make_la_person')
 	buy_sell_modifiers = Service('buy_sell_modifiers')
+	transaction_classification = Service('transaction_classification')
 
-	def __call__(self, data:dict, make_la_person, buy_sell_modifiers):
+	def __call__(self, data:dict, make_la_person, buy_sell_modifiers,transaction_classification):
```

Comment:
This is very minor, but there are several places in this PR where use (lack) of whitespace differs from surrounding code. If possible, could this be standardized?